### PR TITLE
[main] add pd transfer for ascend scheduler

### PR DESCRIPTION
**Pull request by Liccol**
### What this PR does / why we need it?
For offline scenarios, adjust the scheduling process to prioritize the prefill phase of all requests, then process the decode phase of all requests.
### How was this patch tested?

```
max_num_seqs=24,
additional_config={
    "ascend_scheduler_config":{
        "enabled": True,
        "enable_pd_transfer": True,
        "decode_max_num_seqs": 24,
        "enable_chunked_prefill": False
    }
},
```
| input | output | num prompts | max_num_seqs | dp | tp | scheduler      | tps           |
| ------ | ------ | ---------- | ---------------- | ---- | ---- | ---------------- | --------------- |
| dapo-math-17K | 2K   |      384 |             24 |  2 |  1 | v1             |        234.06 |
| dapo-math-17K | 2K   |      384 |             24 |  2 |  1 | pd transfer        | 239.59(+2.4%) |
| dapo-math-17K| 2K   |      384 |             24 |  4 |  1 | v1             |        222.85 |
| dapo-math-17K| 2K   |      384 |             24 |  4 |  1 | pd transfer         |         225.81(+1.3%) |

- vLLM version: v0.10.1.1
- vLLM main: https://github.com/vllm-project/vllm/commit/6fb27881634d89c2e70e9e5fbad1b918c0d916cf

### CODE REVIEW

**Comment by zzzzwwjj** [comment]
Enabling this feature will result differences in definition of `max_num_seqs` between our and the community. Fortunately, disabling this feature will not have any impact. The best solution is to make the community accept our plan.
@wangxiyuan cc

File: vllm_ascend/ops/min_p_logits_processor.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        if self.use_double_tensor:
+            self.min_p_device = torch.empty((max_num_reqs, ),
+                                            dtype=torch.float32,
+                                            device=device)
+        else:
+            self.min_p_device = self.min_p_cpu_tensor
+        self.min_p = self.min_p_device[:0]
+
+
+MinPLogitsProcessor.__init__ = min_p_logits_processor_init_func
```
Comment (wangxiyuan) [inline vllm_ascend/ops/min_p_logits_processor.py]: Do not patch vLLM, you should create ascend `build_logitsprocs` in npu_input_batch instead.

Comment (Liccol) [reply vllm_ascend/ops/min_p_logits_processor.py]: change from patch MinPLogitsProcessor init func to redefine build_logitsprocs, code in [30571f1](https://github.com/vllm-project/vllm-ascend/pull/2753/commits/30571f1a6fdd3e3ca84893eb6e0eaa169574d154)

**Review by wangxiyuan — APPROVED** [review]
